inst/htmlwidgets/lib/leaflet-providers/leaflet-providers.js

```diff
@@ -90,7 +77,7 @@
 
 	L.TileLayer.Provider.providers = {
 		OpenStreetMap: {
-			url: '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+			url: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
```

Comment:
For url changes where only the `https` was dropped, could you please keep it removed? (including Stamen) . I've checked Onemap and they support both http and https. If they could have their protocol dropped as well, that'd be great.

There is a standing bug with RStudio 1.1 (fixed in 1.2 - daily build) that windows and linux can not load https tiles. There is some custom code to make the protocols work if no protocol is provided.  Once 1.2 is released, all custom changes to the protocols will be removed.

Reply:
Onemap http link ex: http://maps-b.onemap.sg/v3/Default/13/6458/4064.png